server/middlewares/frontendMiddleware.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const path = require('path');
+
+// Dev middleware
+const addDevMiddlewares = (app, options) => {
+  const webpackDevMiddleware = require('webpack-dev-middleware');
+  const webpackHotMiddleware = require('webpack-hot-middleware');
```

Comment:
This is included in a function to avoid webpack-dev requires in PROD environment when it might be not installed.